packages/core/saved-objects/docs/openapi/components/examples/export_objects_request.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Export a specific saved object. 
+value:
+  {
+    "objects": [
+      {
+        "type": "index-pattern",
+        "id": "90943e30-9a47-11e8-b64d-95841ca0b247"
+        }
+    ],
+    "includeReferencesDeep": false
```

Comment:
The request body also contains [other optional items](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-export.html#saved-objects-api-export-request-request-body). Our public docs list then under `Request body`.
If the intent with the OpenApi spec is to _replace_ existing public docs, we need to include them too.

Reply:
I believe I've got them all covered in the api@saved_objects@_export.yaml, just not in this particular example. If I've misunderstood and there *are* more properties missing from the spec, it can definitely be improved in subsequent PRs.